bundles/org.openhab.binding.ojelectronics/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -121,13 +121,13 @@
 		<label>Regulation Mode</label>
 		<state readOnly="true">
 			<options>
-				<option value="Auto">Auto</option>
-				<option value="Comfort">Comfort</option>
-				<option value="Manual">Manual</option>
-				<option value="Vacation">Vacation</option>
-				<option value="Frost Protection">Frost Protection</option>
-				<option value="Boost">Boost</option>
-				<option value="Eco">Eco</option>
+				<option value="auto">Auto</option>
```

Comment:
It seems you map to the capitalized values, or am I mistaken?

Reply:
You are right. I have to use lower case letters here and in source code too.